catalog/transformer/catalog-transformer-mgmp/src/main/java/org/codice/alliance/catalog/transformer/mgmp/MgmpConstants.java

```diff
@@ -0,0 +1,151 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.alliance.catalog.transformer.mgmp;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class MgmpConstants {
+
```

Comment:
The other option is making this an interface instead, though that's gone out of style.

A `final` class with a private ctor is good enough.